### DISCUSSION TITLE
Can I port tailwind-elements to Statamic?

### DISCUSSION THREAD

**Discussion by crslp** [mdbootstrap/TW-Elements#1677] [Share ideas & request features]
Hi all

Thanks for this awesome package of tailwind components.
Since I do more and more with Statamic, I would like to ask if it's okay if I take certain tailwind-elements and port them to a Statamic-Content block ?!

I imagine a package that one can install and then, via the cli, publish the content blocks (markup and yaml) to their repo.

In this package, I will give credit to the creators and this repo!

Just want to hear if this is okay :) 
Thanks for feedback!

**Comment by filipkappa** [comment]
Hello Chris,

You can do this, as long as the source code of the project you are porting tailwind-elements is also fully open-source (AGPL).

If that's not the case - you would need to request a custom license here:
https://tailwind-elements.com/license/

Keep coding!
